frontend/public/components/utils/truncate.ts

```diff
@@ -0,0 +1,35 @@
+export enum TruncationPoint {
```

Comment:
Do we expect to have other truncation points in the future? Otherwise, I suggest components just use `truncateMiddle()` directly and then remove `TruncationPoint` / `truncate()`.

Reply:
+1, I'd just leave this as `truncateMiddle`